src/sumo/wrapper/login.py

```diff
@@ -64,20 +74,39 @@ def main():
     env = args.env
     logger.debug("env is %s", env)
 
-    print("Login to Sumo environment: " + env)
+    if args.silent:
+        args.interactive = False
```

Comment:
This feels "wrong" (modifying the args directly) but it may be me who is wrong 🤷